package.json

```diff
@@ -20,28 +20,36 @@
   ],
   "author": "Jakub Strojewski <jakub.strojewski@gmail.com>",
   "license": "WTFPL",
-  "dependencies": {},
+  "dependencies": {
+    "angular": "^1.5.7",
+    "angular-resource": "^1.5.7"
+  },
   "devDependencies": {
-    "bower": "1.7.9",
-    "coveralls": "2.11.9",
-    "del": "2.2.1",
-    "gulp": "3.9.1",
-    "gulp-tslint": "5.0.0",
-    "gulp-typescript": "2.13.6",
-    "jasmine-core": "2.4.1",
-    "karma": "1.1.0",
-    "karma-chrome-launcher": "1.0.1",
-    "karma-coffee-preprocessor": "1.0.0",
-    "karma-coverage": "1.0.0",
-    "karma-jasmine": "1.0.2",
-    "karma-mocha-reporter": "2.0.4",
-    "karma-phantomjs2-launcher": "0.5.0",
-    "merge-stream": "1.0.0",
-    "phantomjs2": "2.2.0",
-    "require-dir": "0.3.0",
-    "run-sequence": "1.2.1",
-    "tslint": "3.12.1",
-    "typescript": "1.8.10",
-    "typings": "1.3.1"
+    "angular-mocks": "^1.5.7",
+    "babel-core": "^6.10.4",
+    "babel-preset-es2015": "^6.9.0",
```

Comment:
please remove babel - one of my main goal here is not to use any libs if they are not esential